ga4gh/datamodel/ontologies.py

```diff
@@ -5,25 +5,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import fnmatch
-import os
 import ga4gh.protocol as protocol
 
 
-class FileSystemOntology(object):
+class Ontology(object):
```

Comment:
No, it's actually not the ontology, a lot of information is
missing. The ontology has a hierarchal structure.  The ontology
is on a format like OBO:

   http://www.sequenceontology.org/resources/ontology.html

This just gives the ability to take a term name in the variant
annotations inside of a VCF and map it to the full ontology term
which provides a reference that can be use to query an ontology
server.

don't worry, we haven't seen that last of ontologies.


Jerome Kelleher <notifications@github.com> writes:
> In ga4gh/datamodel/ontologies.py:
> 
> >  import ga4gh.protocol as protocol
> >
> >
> > -class FileSystemOntology(object):
> > +class Ontology(object):
> 
> I'm confused about this @diekhans --- we have a text file that contains an
> Ontology (which, at the moment corresponds to a mapping between IDs and terms),
> right? Which we call (say) "sequence_ontology". Where does the 'map' come into
> it? I'm happy to change it back, I'm just confused.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>